lib/libutils/ext/sub.mk

```diff
@@ -12,7 +12,11 @@ srcs-y += fault_mitigation.c
 srcs-y += qsort_helpers.c
 srcs-y += array.c
 srcs-y += base64.c
-
+srcs-$(call cfg-one-enabled,CFG_TA_SANITIZE_UNDEFINED \
+			    CFG_CORE_SANITIZE_UNDEFINED) += ubsan.c
+cflags-remove-ubsan.c-$(call cfg-one-enabled,CFG_TA_SANITIZE_UNDEFINED \
```

Comment:
Please use `cflags-remove-ubsan.c-y` to keep it simple.

Reply:
Done